Configuration.props

```diff
@@ -31,6 +31,7 @@
     <MonoSgenBridgeVersion Condition=" '$(MonoSgenBridgeVersion)' == '' ">5</MonoSgenBridgeVersion>
     <HOME Condition=" '$(HOME)' == '' ">$(HOMEDRIVE)$(HOMEPATH)</HOME>
     <AndroidApiLevel Condition=" '$(AndroidApiLevel)' == '' ">25</AndroidApiLevel>
+    <AndroidPreviousFrameworkVersion>v1.0</AndroidPreviousFrameworkVersion>
```

Comment:
I guess it should. Command line /p: overrides it anyway, but I guess we want Configuration.Override.props to a have a chance as well :-)